secrets/secrets.go

```diff
@@ -131,6 +131,7 @@ func (k *Keeper) Decrypt(ctx context.Context, ciphertext []byte) (plaintext []by
 //
 // ErrorAs panics if i is nil or not a pointer.
 // ErrorAs returns false if err == nil.
+// See https://godoc.org/gocloud.dev#As for background information.
```

Comment:
Since this is the only `As` function in this package, it should have the full comment about seeing examples etc. in addition to background information.

Reply:
We don't have examples for secrets ErrorAs yet... otherwise I'll update it to be more consistent